README.md

```diff
@@ -128,6 +129,8 @@ For M1 Macs, here's the following steps needed
 - Step 2: export LIBRARY_PATH=/opt/homebrew/lib
 - Step 3: Follow the steps as per the docs.
 
+On Ubuntu, install `libhwloc-dev`.
```

Comment:
This should be in the same line as `- [hwloc](https://www.open-mpi.org/projects/hwloc/)`